composable_source/readers/med_mentions_reader.py

```diff
@@ -65,6 +65,7 @@ def _parse_pack(self, collection: str) -> Iterator[DataPack]:
         logging.info("Processing %s.", collection)
         doc = open(collection, "r", encoding="utf8")
         text, pack_name = "", ""
+        pack: DataPack = DataPack()
```

Comment:
It seems the pack is created twice. Do we have to do this?

Reply:
I checked the sample test file, this line is needed. In this function it is trying to create multiple datapacks, the logic is in https://github.com/petuum/composing_information_system/blob/main/composable_source/readers/med_mentions_reader.py#L69-L77